packages/contact-manager-rest-api/src/contact-manager-api-server.ts

```diff
@@ -0,0 +1,85 @@
+import { agentContext } from '@sphereon/ssi-sdk.core'
+import { TAgent } from '@veramo/core'
+
+import express, { Express, Router } from 'express'
+import { IContactManagerAPIEndpointOpts, IRequiredPlugins } from './types'
+import {
+  identityReadEndpoints,
+  partiesReadEndpoint,
+  partyDeleteEndpoint,
+  partyReadEndpoint,
+  partiesTypeReadEndpoint,
+  partyWriteEndpoint,
+  partyTypeReadEndpoint,
+  identitiesReadEndpoint
+} from './api-functions'
+import { copyGlobalAuthToEndpoints, ExpressSupport } from '@sphereon/ssi-express-support'
+
+export class ContactManagerApiServer {
+  private readonly _express: Express
+  private readonly _agent: TAgent<IRequiredPlugins>
+  private readonly _opts?: IContactManagerAPIEndpointOpts
+  private readonly _router: Router
+
+  constructor(args: { agent: TAgent<IRequiredPlugins>; expressSupport: ExpressSupport; opts?: IContactManagerAPIEndpointOpts }) {
+    const { agent, opts } = args
+    this._agent = agent
+    copyGlobalAuthToEndpoints({ opts, keys: ['partyRead', 'partyWrite', 'partyTypeRead', 'identityRead'] })
+    if (opts?.endpointOpts?.globalAuth?.secureContactManagerEndpoints) {
+      copyGlobalAuthToEndpoints({ opts, keys: ['partyRead', 'partyWrite', 'partyTypeRead', 'identityRead'] })
+    }
+    const enableFeatures = opts?.enableFeatures ?? [
+      'party_read',
+      'party_write',
+      'party_type_read',
+      'identity_read',
+    ]
+    this._opts = opts
+    this._express = args.expressSupport.express
+    this._router = express.Router()
+    const context = agentContext(agent)
+    const features = enableFeatures ?? [
```

Comment:
What's up with this. Above you are already doing enabledFeatures with a nullish coalescence. This one will never be triggered since it cannot be null at this point. Also why would the defaults differ between the two?